feel/feeldiscr/operatorinterpolation.hpp

```diff
@@ -1074,30 +1258,26 @@ OperatorInterpolation<DomainSpaceType, ImageSpaceType,IteratorRange,InterpType>:
 
                                 for ( uint16_type jloc = 0; jloc < domain_basis_type::nLocalDof; ++jloc )
                                 {
-                                    uint16_type compDomain = (domain_basis_type::is_product)? comp : 0;
+                                    uint16_type nCompDomain = (domain_basis_type::is_product)?domain_basis_type::nComponents:1;
+                                    for(int cdomain = 0;cdomain < nCompDomain;++cdomain )
+                                    {
```

Comment:
Actually it could depending on the expression. 
But I agree it needs to be optimized. 
I will change back in the Lagrange case to have a marching like before. 
However the changes were necessary to support the various operators of de rahm